dipy/align/tests/test_imwarp.py

```diff
@@ -50,69 +50,69 @@ def test_diffeomorphic_map_2d():
     np.random.seed(2022966)
     domain_shape = (10, 10)
     codomain_shape = (10, 10)
-    #create a simple affine transformation
+    # create a simple affine transformation
     nr = domain_shape[0]
     nc = domain_shape[1]
     s = 1.1
     t = 0.25
-    trans = np.array([[1, 0, -t*nr],
-                      [0, 1, -t*nc],
+    trans = np.array([[1, 0, -t * nr],
+                      [0, 1, -t * nc],
                       [0, 0, 1]])
     trans_inv = np.linalg.inv(trans)
-    scale = np.array([[1*s, 0, 0],
-                      [0, 1*s, 0],
+    scale = np.array([[1 * s, 0, 0],
+                      [0, 1 * s, 0],
                       [0, 0, 1]])
     gt_affine = trans_inv.dot(scale.dot(trans))
 
-    #create the random displacement field
+    # create the random displacement field
     domain_grid2world = gt_affine
     codomain_grid2world = gt_affine
     disp, assign = vfu.create_random_displacement_2d(
-                        np.array(domain_shape, dtype=np.int32),
-                        domain_grid2world,np.array(codomain_shape,
-                                                   dtype=np.int32),
-                        codomain_grid2world)
+        np.array(domain_shape, dtype=np.int32),
+        domain_grid2world, np.array(codomain_shape,dtype=np.int32),
```

Comment:
This is much more readable! it just needs a space after the comma:
```
domain_grid2world, np.array(codomain_shape, dtype=np.int32),
```
and remove unused line 3:
```
 import numpy.testing as npt
```
This is what `flake8` reports on `test_imwarp.py`:
```
test_imwarp.py:3:1: F401 'npt' imported but unused
test_imwarp.py:72:51: E231 missing whitespace after ','
```